po/es.po

```diff
@@ -6,20 +6,20 @@ msgstr ""
 "PO-Revision-Date: 2021-09-14 15:25\n"
 "Last-Translator: \n"
 "Language-Team: Spanish\n"
-"Language: es_ES\n"
+"Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Crowdin-Project: clapper\n"
 "X-Crowdin-Project-ID: 473374\n"
-"X-Crowdin-Language: es-ES\n"
+"X-Crowdin-Language: es_ES\n"
```

Comment:
Same here, do not change autogenerated parts, unless there is a reason to do so (which would need changes at the site where this file is autogenerated). If there is a reason, please speak up.